src/ContributionsFromRenee/SokoBox.js

```diff
@@ -0,0 +1,62 @@
+//copied from ./entity/SokoBox.js
+import 'phaser';
+
+export default class SokoBox extends Phaser.Physics.Arcade.Sprite {
+  constructor(scene, x, y, spriteKey) {
+    super(scene, x, y, spriteKey);
+    this.scene = scene;
+    this.scene.add.existing(this);
+    this.scene.physics.world.enable(this);
+    this.scene.events.on('update', this.update, this)
+  } 
+
+  update() {
+    this.updateMovement()
+  }
+
+  updateMovement() {
+    if (this.body.touching.right) {
+      this.body.setImmovable(false)
+      this.body.setVelocityX(-80)
+      this.body.setVelocityY(0)
+      this.body.setBounce(0)
```

Comment:
This looks pretty repetitive. I would make a helper function for this repetition. You only change like 1 or 2 values.

Also there is a concept called "magic numbers" that devs try to avoid. If 80 is the magic number, set it to a variable and list it at the top of the file like `const VELOCITY = 80`